src/gui/footer/footer_item_z_heigth.cpp

```diff
@@ -0,0 +1,27 @@
+#include "footer_item_z_heigth.hpp"
+#include "marlin_client.h"
+#include "resource.h"
+#include <algorithm>
+#include <cmath>
+
+string_view_utf8 FooterItemZHeigth::static_makeView(float value) {
+    static std::array<char, 7> buff;
+    float value_clamped = std::clamp((float)value, 0.f, 100.f);
+
+    int printed_chars = snprintf(buff.data(), buff.size(), "%.2f", (double)value_clamped);
+
+    if (printed_chars < 1) {
+        buff[0] = '\0';
+    } else {
+        while ((--printed_chars) > 2 && buff[printed_chars] == '0' && buff[printed_chars - 1] != '.') {
```

Comment:
I'm not sure about operator `>` `&&` and `==` priority
please consider adding brackets

Reply:
👍